packages/copy-to-clipboard/package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "@cloud-ru/frontend-tools-copy-to-clipboard",
```

имена соседних пакетов посмотри, тут тоже "короткая версия" должна быть, как и в "локальных" версиях пакетов

```suggestion
  "name": "@cloud-ru/ft-copy-to-clipboard",
```